<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPreferences</class>
 <widget class="QWidget" name="GeneralPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>General settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Preferred GUI language</string>
          </property>
          <property name="buddy">
           <cstring>guiLanguage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="guiLanguage">
          <property name="toolTip">
           <string>The GUI will use this language whenever possible. English will be used as a fallback.</string>
          </property>
          <item>
           <property name="text">
            <string>System default</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Preferred dictionary language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dictLanguage">
          <property name="toolTip">
           <string>The dictionaries will use this language whenever possible. English will be used as a fallback.</string>
          </property>
          <item>
           <property name="text">
            <string>Same as GUI</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>You need to restart Tagaini Jisho for these changes to take effect.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="datesGroupBox">
     <property name="title">
      <string>Dates</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Weeks start on</string>
        </property>
        <property name="buddy">
         <cstring>firstDayOfWeek</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="firstDayOfWeek"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="updatesGroupBox">
     <property name="title">
      <string>Updates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkForUpdates">
          <property name="text">
           <string>Check for updates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="checkInterval">
          <property name="specialValueText">
           <string>every day</string>
          </property>
          <property name="suffix">
           <string> days</string>
          </property>
          <property name="prefix">
           <string>every </string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkForBetaUpdates">
        <property name="text">
         <string>Also check for development versions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nextCheck">
        <property name="text">
         <string>Next check:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedGroupBox">
     <property name="title">
      <string>Entries cache (advanced)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="cacheSizeDefault">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cacheSize">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Defines how many recent entries are kept in memory. High values may improve performance when manipulating a large number of entries.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>guiLanguage</tabstop>
  <tabstop>firstDayOfWeek</tabstop>
  <tabstop>checkForUpdates</tabstop>
  <tabstop>checkInterval</tabstop>
  <tabstop>checkForBetaUpdates</tabstop>
  <tabstop>cacheSizeDefault</tabstop>
  <tabstop>cacheSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkForUpdates</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkForBetaUpdates</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
